server/routerlicious/packages/routerlicious-base/src/riddler/runnerFactory.ts

```diff
@@ -39,6 +40,13 @@ export class RiddlerResourcesFactory implements utils.IResourcesFactory<RiddlerR
         const tenantsCollectionName = config.get("mongo:collectionNames:tenants");
         const secretManager = new services.SecretManager();
 
+        let secretLength: number = config.get("auth:secretLength");
+        if (!secretLength) {
+            secretLength = 31;
+        } else {
+            secretLength = Math.min(secretLength, 31);
+        }
```

Comment:
It might be kind to notify the user they've exceeded the secretLength (e.g., via the log, console.warn, or whatever our convention is.)

Reply:
Yes. This code will be used by a tool (cli or web app). We will add a warning there.